sales/sale_amazon/apply.rst

```diff
@@ -0,0 +1,124 @@
+information** section, select the correct region of your seller account. For the other sections,
+adapt your responses in accordance with your business case.
+
+Give a particular attention to the **Merchant Fulfilled Shipping** function. It should only be
+checked if you ship your products yourself. You should uncheck it if you sell exclusively with the
+*Fulfillment by Amazon* service. Please note that requesting this function is not recommended if you
+are hosted on *Odoo Online* or *Odoo.sh* as the additional security requirements asked by Amazon may
+not be met by Odoo.
+
+.. warning ::
+   Depending on several factors (your region, whether you checked the **Merchant Fulfilled
+   Shipping** function (i.e. you request access to Personally Identifiable Information of your
+   customers), etc.), Amazon may request you to fill out a second form before granting you MWS
+   access keys. As that form depends on the data protection policy of the region of your seller
+   account (e.g. GDPR in Europe), we cannot provide you with a pre-filled form. Instead, the answers
+   of questions related to Odoo are listed in the `Answer the Additional Form`_ section.
+
+.. image:: ./media/dev_form.png
+
+Answer the Additional Form
+==========================
+
+This section lists all questions asked by Amazon in additional forms. The answers are tailored for
+*Odoo Online* and *Odoo.sh*. If you did not receive any additional form after your :ref:`initial
+application for Amazon MWS keys <amazon/developer-form>`, disregard the following.
+
+
+.. warning::
+   If you host Odoo yourself (*on-premise*), adapt the answers related to hosting to your own
+   infrastructure and data protection policy.
+
+
+- **Describe all functionalities in your application where Personally Identifiable Information (e.g.
+  customer name, street address, billing address) is required.**
+
+  - If you did not apply for the *Merchant Fulfilled Shipping* function:
+
+    \- Generation of customer invoices
+
+  - If you applied for the *Merchant Fulfilled Shipping* function:
+
+    | - Generation of delivery orders
+    | - Generation of customer invoices
+
+- **List all outside parties with whom your organization shares Amazon Information (e.g. information
+  exposed by Amazon through Amazon MWS, Seller Central, or Amazon's public-facing websites) and
+  describe how your organization shares this information.**
+
+  - If you do not share Amazon Information with outside parties:
+
+    Odoo does not share any information with outside parties.
+
+  - If you share Amazon Information with outside parties:
+
+    [Description of your organization's policy regarding Amazon Information]
+
+- **List all non-Amazon MWS sources where you retrieve Amazon Information.**
+
+  Odoo only relies on MWS to retrieve Amazon Information.
+
+- **Describe how your organization restricts public access to databases, file servers, and
+  desktop/developer endpoints.**
+
+  TODO
```

Comment:
To check with saas office:
- http access to postgres is disabled and standard ports are closed, the db is only accessible through a socket on the server itself
- the reverse proxy only serves whitelisted directories that are from sources only controlled by Odoo S.A.
- API endpoints are password protected (PBKDF2+SHA512 encryption, salted + stretched for thousands of rounds)
- Login credentials are always transmitted securely over HTTPS